src/prefect/cli/get.py

```diff
@@ -0,0 +1,423 @@
+)
+@click.option("--project", "-p", help="The name of a project to query.", hidden=True)
+@click.option(
+    "--limit", "-l", default=10, help="A limit amount of tasks to query.", hidden=True
+)
+def tasks(name, flow_name, flow_version, project, limit):
+    """
+    Query information regarding your Prefect tasks.
+
+    \b
+    Options:
+        --name, -n          TEXT    A task name to query
+        --flow-name, -fn    TEXT    A flow name to query
+        --flow-version, -fx INTEGER A flow version to query
+        --project, -p       TEXT    The name of a project to query
+        --limit, -l         INTEGER A limit amount of tasks to query, defaults to 10
+    """
+
+    query = {
+        "query": {
+            with_args(
+                "task",
+                {
+                    "where": {
+                        "_and": {
+                            "name": {"_eq": name},
+                            "flow": {
+                                "name": {"_eq": flow_name},
+                                "project": {"name": {"_eq": project}},
+                                "version": {"_eq": flow_version},
+                            },
+                        }
+                    },
+                    "limit": limit,
+                    "order_by": {"created": EnumValue("desc")},
+                },
+            ): {
+                "name": True,
+                "created": True,
+                "flow": {"name": True, "version": True},
+                "mapped": True,
+                "type": True,
+            }
+        }
+    }
+
+    result = Client().graphql(query)
+
+    task_data = result.data.task
+
+    output = []
+    for item in task_data:
+        output.append(
+            [
+                item.name,
+                item.flow.name,
+                item.flow.version,
+                pendulum.parse(item.created).diff_for_humans(),
+                item.mapped,
+                item.type,
+            ]
+        )
+
+    click.echo(
+        tabulate(
+            output,
+            headers=["NAME", "FLOW NAME", "FLOW VERSION", "AGE", "MAPPED", "TYPE"],
+            tablefmt="plain",
+            numalign="left",
+            stralign="left",
+        )
+    )
+
+
+@get.command(hidden=True)
+@click.option(
+    "--name", "-n", required=True, help="A flow run name to query", hidden=True
+)
+@click.option(
+    "--info", "-i", is_flag=True, help="Retrieve detailed logging info", hidden=True
+)
+def logs(name, info):
+    """
+    Query logs for a flow run.
+
+    \b
+    Options:
+        --name, -n      TEXT    A flow run name to query        [required]
+        --info, -i              Retrieve detailed logging info
+    """
+    log_query = {
+        with_args("logs", {"order_by": {EnumValue("timestamp"): EnumValue("asc")}}): {
+            "timestamp": True,
+            "message": True,
+            "level": True,
```

Comment:
Could you include lineno in this?

Reply:
The line number isn't immediately available as a field and is nested in the `info`